tests/system/testdata/authz/default/projects.txtar

```diff
@@ -0,0 +1,111 @@
+user zumi
+
+# create a project for zumi in zumi's personal org.
+ak project create --name pz
+return code == 0
+
+# make sure zumi is a member of zumi's personal org
+ak project get pz -f
+return code == 0
+output equals_jq .project.org_id $ZUMI_OID
+capture_jq pz_pid .project.project_id
+
+ak project get zumi_org.pz
+return code == 0
+
+user gizmo
+
+# create a project for gizmo in cat's org.
+ak project create --name pg --org $CATS_OID
+return code == 0
+capture_jq pg_pid .project_id
+
+ak project get $pg_pid -f
+return code == 0
+output equals_jq .project.org_id $CATS_OID
+
+# zumi should be able to see gizmo's project
+user zumi
+ak project get $pg_pid -f
+return code == 0
+
+# gizmo should not be able to see zumi's project
+user gizmo
+ak project get $pz_pid -f
+return code == $RC_NOT_FOUND
+
+# gizmo cannot create a project on zumi's org
+user gizmo
+ak project create --org $ZUMI_OID
+return code == $RC_UNAUTHZ
+
+# zumi sees only her projects by default
+user zumi
+ak project list
+return code == 0
+output equals_jq .[].name pz
+
+# zumi cannot see gizmo's projects
+user zumi
+ak project list --org $GIZMO_OID
+return code == $RC_UNAUTHZ
```

Comment:
Shouldn't list for someone else's stuff also return NOT FOUND instead of UNAUTHZ?